Gemfile

```diff
@@ -9,7 +9,7 @@ gem 'airbrake', '~> 4.2.0'
 gem 'gds-sso', '~> 11.0.0'
 gem 'gds-api-adapters', '24.3.0'
 
-gem 'govuk_admin_template', '~> 2.3.1'
+gem 'govuk_admin_template', '3.0.0'
```

Comment:
Almost every other app is wrong. 

https://github.com/alphagov/styleguides/blob/bebfb81551f900dc6be623f508f2b19fa8032ce1/ruby.md#gemfile